src/styles.css

```
@@ -24,10 +24,32 @@
   --skeleton-base:#2a2a3a;--skeleton-shine:#33334a;
 }
 html{scroll-behavior:smooth}
-body{background:var(--bg);color:var(--ink);font-family:'Plus Jakarta Sans',sans-serif;min-height:100vh;overflow-x:hidden;transition:background .3s,color .3s}
+body{
+  background:
+    radial-gradient(circle at top left, rgba(249,115,22,.08), transparent 30%),
```

Stylelint (17.10.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)